paintBrush.js

```diff
@@ -66,3 +66,40 @@ function startPainting() {
     painting = true;
 }
 
+//story 7
+document.querySelector('form').addEventListener('submit', e => {
+
+    e.preventDefault()
+    // created a variable to contain the users text input
+    let text = document.querySelector('input').value
+    ctx.font = '50px "Hiragino Maru Gothic Pro"'
+    //create a fill text function that places the users text input at a set
+    //place on the canvas
+    ctx.fillText(text, 10, 50)
+})
+
+//when the text button is clicked, it should reveal the text input
+document.querySelector('.text').addEventListener('click', e => {
+    e.preventDefault()
+    document.querySelector('#text').setAttribute('type', 'text')
+    document.querySelector('#submit').setAttribute('type', 'submit')
+})
+// story 9
+document.querySelector('.print').addEventListener('click', e => {
```

Comment:
For some reason, this doesn't work for me at all on story 9. 

It works nicely on s9t2, but something in this branch is breaking it for me